sdk/docs/pages/zksend/dapp.mdx

```diff
@@ -4,43 +4,43 @@ import { Callout } from 'nextra/components';
 
 <Callout type="info">
 	If you are using `dapp-kit`, you do not need to install any additional packages to integrate with
-	the zkSend wallet. [Read how to integrate with dapp-kit.](../dapp-kit/zksend.mdx)
+	the Stashed wallet. [Read how to integrate with dapp-kit.](../dapp-kit/stashed.mdx)
 </Callout>
 
-Using the zkSend SDK, you can allow users to connect to the zkSend wallet from your dApp. The wallet
-is provided through the [Wallet Standard](https://docs.sui.io/standards/wallet-standard), and should
-appear automatically in your existing wallet connection UI.
+Using the zkSend SDK, you can allow users to connect to the Stashed wallet from your dApp. The
+wallet is provided through the [Wallet Standard](https://docs.sui.io/standards/wallet-standard), and
+should appear automatically in your existing wallet connection UI.
 
 ## Considerations
 
-- zkSend only supports mainnet at this time.
-- Users will only be able to connect their zkLogin wallet from zkSend.
-- The zkLogin account will be managed by zkSend, you do not need to set up any OAuth providers.
+- Stashed only supports mainnet at this time.
```

Comment:
```suggestion
- Stashed only supports Mainnet at this time.
```
Style guide states this form, but better to be consistent with rest of this site if this is the way it's done.